src/po/gmoccapy/de.po

```diff
@@ -1917,8 +1917,9 @@ msgid "Rot"
 msgstr "Rot"
 
 #: ../../../../lib/python/gladevcp/offsetpage.glade.h:14
+#, fuzzy
 msgid "Rotation of Z"
-msgstr "Drehung um Z"
```

Comment:
> @smoe I don't agree with XY-Drehung and Arbeitsoffset. Lathe Wear Offsets is more like Werkzeugverschleiß. Rotation of Z I would translate as "Rotation um Z"

It seems a bit like a bug in Weblate. I changed this string at the same session where I also made the other changes.
(Look at the time and author in the header) But this change was overwritten again by the old one. May Steffen was logged in in Weblate at the same time?

Reply:
Just correct any nonsense of mine, please. Literally the "Rotation of Z" is more like a "Rotation von Z", i.e. a change of direction of Z while keeping the unit length. I admit that I do not know how this makes any sense if X and Y do not turn with it. Weblate finds it on https://hosted.weblate.org/translate/linuxcnc/linuxcnc/de/?q=Drehung&sort_by=-priority%2Cposition&offset=14 . "Drehung um Z" würde "Rotation around Z" im Englischen sein?!?

Reply:
I admit not to have any idea what an XY-Rotation from https://hosted.weblate.org/translate/linuxcnc/linuxcnc/de/?offset=1&q=XY-Drehung&sort_by=-priority%2Cposition&checksum= could possibly be if not an XY-Drehung :) Again, please override.

Reply:
I removed the Arbeitsoffsets. Abnutzungs-Kompensation? Anyway - someone will know the right word.

Reply:
This is actually what "Rotation of Z" is doing. Here a value of 45 degrees:

![grafik](https://user-images.githubusercontent.com/67957916/149627451-d568ac2e-500b-484d-9fe5-1f73a7fd059a.png)